drivers/disk/nvme/nvme_namespace.h

```diff
@@ -155,7 +155,7 @@ void nvme_namespace_data_swapbytes(struct nvme_namespace_data *s)
 
 struct nvme_namespace {
 	struct nvme_controller *ctrlr;
-	struct nvme_namespace_data data __aligned(0x1000);
+	struct nvme_namespace_data data;
```

Comment:
Not sure why this was needed and now isn't? Update the base commit to not have it